### PR TITLE
update websecurity.md

### PR DESCRIPTION
**Pull request by Christiandike**
- fix typos/edit text
- add important links

### CODE REVIEW

**Comment by Christiandike** [comment]
Hello @sgibson91 , please could you review this PR? I would really appreciate your time. Thank you 🙏

File: docs/source/reference/websecurity.md [hunk 1]
```diff
@@ -118,15 +118,20 @@ extend to securing the `tcp` sockets as well.
...
+<<<<<<< HEAD
+We recommend that you do periodic reviews of your deployment's security. It's
+good practice to keep [JupyterHub](https://readthedocs.org/projects/jupyterhub/), [configurable-http-proxy](https://github.com/jupyterhub/configurable-http-proxy), and [nodejs
+versions](https://github.com/nodejs/Release) up to date.
+=======
 We recommend that you do periodic reviews of your deployment's security. It is
 good practice to keep JupyterHub, configurable-http-proxy, and nodejs
 versions up to date.
 
+> > > > > > > main
```
Comment (sgibson91) [inline docs/source/reference/websecurity.md:130]: This looks like the remain of a merge conflict.

The following lines should be removed:

```
<<<<<<< HEAD
=======
> > > > > > > main
```

Can you then combine the two sentences that begin "We recommend that..." into a single sentence please?

**Comment by Christiandike** [comment]
Thank you @sgibson91 
I've applied the changes. I don't know how I completely missed that. Thanks again!

File: docs/source/reference/websecurity.md [hunk 2]
```diff
@@ -118,15 +118,13 @@ extend to securing the `tcp` sockets as well.
...
-good practice to keep JupyterHub, configurable-http-proxy, and nodejs
-versions up to date.
+We recommend that you do periodic reviews of your deployment's security. It's
+good practice to keep [JupyterHub](https://readthedocs.org/projects/jupyterhub/), [configurable-http-proxy](https://github.com/jupyterhub/configurable-http-proxy), and [nodejs
+versions](https://github.com/nodejs/Release) up to date.
 
 A handy website for testing your deployment is
 [Qualsys' SSL analyzer tool](https://www.ssllabs.com/ssltest/analyze.html).
 
-[configurable-http-proxy]: https://github.com/jupyterhub/configurable-http-proxy
```
Comment (minrk) [inline docs/source/reference/websecurity.md:128]: This link target was used, please don't remove it.

You can also re-use the same link above with `[configurable-http-proxy][]`.

**Comment by Christiandike** [comment]
Thank you @minrk 
I've fixed it

**Review by sgibson91 — APPROVED** [review]
Thank you @Christiandike - Don't forget to move your handle to the "Done" column of the relevant spreadsheet now!

**Pull request merged**